CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Change Log
+
+## [v0.1.0](https://github.com/auth0/auth0-spa-js/tree/v0.1.0) (2020-07-31)
+
+**Early Access Release**
+Do not use it in a Production environment.
+
+### Installation
+
+In order to install this package on your application, download the `auth0-auth0-angular-0.1.0.tgz` file from the Releases section on the Github repository and run the following command:
+
+```bash
+npm install /path/to/auth0-auth0-angular-0.1.0.tgz
+```
```

Comment:
What is this content and where has it come from?

It's just it has references to SPA SDK and the content doesn't look like a typical changelog

Reply:
Good 👀 . I needed to have something there before for the changelog script to work, so I added the first release and some notes. I'll fix the link to point to this repo.